### PR TITLE
[newrelic-pixie] template the cluster registration wait image

### PR DESCRIPTION
**Pull request by kpattaswamy**
#### Is this a new chart
No
#### What this PR does / why we need it:
Templates the default cluster-registraton-wait image.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/newrelic-pixie/values.yaml [hunk 1]
```diff
@@ -23,6 +23,12 @@
 # The key in the customSecretApiKeyName secret that contains the Pixie API key
 # customSecretApiKeyKey:
 
+clusterRegistrationWaitImage:
+  repository: gcr.io/pixie-oss/pixie-dev-public/curl
+  tag: "1.0"
+  pullPolicy: IfNotPresent
+  pullSecrets: []
```
Comment (benkilimnik) [inline charts/newrelic-pixie/values.yaml:30]: where do the `clusterRegistrationWaitImage.pullSecrets` get used?

**Comment by benkilimnik** [comment]
This can be done in a separate PR, but could we update the `newrelic-pixie` chart readme with the new configuration options for the cluster-reg-wait image?

File: charts/newrelic-pixie/values.yaml (hunk 1 above)
Comment (kpattaswamy) [reply charts/newrelic-pixie/values.yaml]: Removed, was stale from the other image spec.

**Comment by kpattaswamy** [comment]
@benkilimnik Yeah agreed, the readme and Chart.yaml have not been up to date, will likely involve some bigger changes to refactor

**Review by benkilimnik — APPROVED** [review]

**Pull request merged**